### ISSUE TITLE
Unable to load whisperer ggml-tiny.en.bin in ubuntu 22.04

### ISSUE THREAD

**Issue by Idimmusix** [occ-ai/obs-localvocal#69]
Hi, OBS crashes whenever I try to add LocalVocal to my audio input.
I have tried creating a new audio input from sources and also using the audio input from the default Audio Mixer section, same issue.

Is there something I'm missing?

## Steps to reproduce the error.
 - Download the Obs-localvocal from the [releases](https://github.com/occ-ai/obs-localvocal/releases/download/0.1.0/obs-localvocal-0.1.0-x86_64-linux-gnu.deb)
 - Install using `dpkg -i obs-localvocal-0.1.0-x86_64-linux-gnu.deb`
 - Open obs-studio 30.0.1
 - Click the 3 dots under Mic/Aux in the Audio Mixer section
![Screenshot from 2024-02-24 11-33-20](https://github.com/occ-ai/obs-localvocal/assets/88910783/2316b125-6a22-416c-af0c-7c7b88b38fd3)
 - Select Filters
![Screenshot from 2024-02-24 11-35-55](https://github.com/occ-ai/obs-localvocal/assets/88910783/1bf6e941-c4a1-4c06-ba83-ee259b3f6b31)
 - Click on the + and select the LocalVocal Transcription
 
 Obs Crashes

While trouble-shooting initially, I removed the ggml-tiny.en.bin from the `/usr/share/obs/obs-plugins/obs-localvocal/models/` directory, when I tried again, it downloaded a new one to the `~/.config/obs-studio/plugin_config/obs-localvocal/models/` but still crashed after download

Here's the report.

```
11:21:30.968: [obs-localvocal] Checking if model models/ggml-tiny.en.bin exists in data...
11:21:30.968: [obs-localvocal] Model models/ggml-tiny.en.bin not found in data.
11:21:30.968: [obs-localvocal] Model path in config: /home/idimma/.config/obs-studio/plugin_config/obs-localvocal/models/ggml-tiny.en.bin
11:21:30.968: [obs-localvocal] Model exists in config folder: /home/idimma/.config/obs-studio/plugin_config/obs-localvocal/models/ggml-tiny.en.bin
11:21:30.968: [obs-localvocal] Loading whisper model from /home/idimma/.config/obs-studio/plugin_config/obs-localvocal/models/ggml-tiny.en.bin
```

You can view the full file here
[obs-crash-report.txt](https://github.com/occ-ai/obs-localvocal/files/14392512/obs-crash-report.txt)

**Comment by royshil** [comment]
should be fixed by #94

**Comment by royshil** [comment]
stale
